doc/STARTING_GUIDE.md

```diff
@@ -16,7 +16,26 @@ There are three steps necessary to start development, in this case for Android:
 
 1. `make run-clojure` - Compiles Clojure into JavaScript, watches for changes on cljs files, and hot-reloads code in the app
 2. `make run-metro` - Starts metro bundler and watches JavaScript code
-3. `make run-android` - Builds the Android app and starts it on the device
+3. `make run-android` - Builds the Android app and starts it on the device.
+
+> If you're on Mac and wish to develop for iOS you can use:
+
+- `make run-ios`
+
+> You can specify the simulator type by adding the SIMULATOR flag like this
+
+- `make run-ios SIMULATOR="iPhone 11 Pro"`
+
+> If you wish to work on a physical iPhone:
+
+1. You must have your device UDID registered on our development team on Apple.
+2. To get your UDID, Connect your iPhone to your Mac.
+3. Open Finder on your Mac, Navigate to your iPhone and right click your UDID and press copy.
```

Comment:
There's plenty of website out there explaining how to get the UDID, like for example:
https://www.itexico.com/blog/find-unique-device-identifier-udid-on-the-iphone

Link it in the text rather than writing it all out. People don't read walls of text so keep it short and sweet.

Reply:
Thanks Jakub, Can you check again please ?